full-service/src/json_rpc/v2/api/response.rs

```diff
@@ -102,6 +102,7 @@ pub enum JsonCommandResponse {
     },
     get_address_details {
         details: PublicAddress,
+        short_address_hash: String,
```

Comment:
I'm a bit on the fence with this suggestion

My initial reaction to this field name is that `address` is redundant.

Seeing the type `ShortAddressHash` later in the code, I can understand how this field name may have been chosen for consistency.  I think the use of `short_address_hash` is falling more into the camp of using the type name in the field/variable, than naming the field based on what it means to the containing struct.
```suggestion
        short_hash: String,
```
consider `address.short_address_hash` versus `address.short_hash`

if this change is made the field in verify_address should be renamed as well.

Reply:
thanks. decided to go with `address_hash` instead as `short` too was redundant (it referred to the length of the hash, which is self-evident) and just `hash` seems too generic.